tasks/seedManagement.js

```diff
@@ -47,31 +47,8 @@ task("sendTransactionSeed", "send transaction to Gnosis Safe")
       (trx.gasPrice = 0),
       (trx.nonce = await gnosis.getCurrentNonce());
 
-    await signerInstance.once("SignatureCreated", async (signature, hash) => {
```

Comment:
we don't listen to the `SignatureCreated` event anymore, because further down we use the `.wait` method?:

```ts
      await signerInstance.generateSignature(
      )
    )
      .wait()
      .then(async () => {
```

Reply:
Yes indeed, this is the pattern used in the `lbpmanagerManagement.js` as well